### PR TITLE
Logging with loguru in place of using print statements 

### PR DESCRIPTION
**Pull request by conradstorz**
I have added logfiles and console output using the Loguru module. This branch is tested and currently working for me.

### CODE REVIEW

**Comment by TheLastGimbus** [comment]
!time

**Comment by TheLastGimbus** [comment]
!time

File: google_photos_takeout_helper/__main__.py [hunk 1]
```diff
@@ -1,3 +1,13 @@
+from loguru import logger
+import sys
+
+logger.remove() # removes the default console logger provided by Loguru.
+# I find it to be too noisy with details more appropriate for file logging.
+logger.add(sys.stdout, format="{message}", level="INFO") # INFO and messages of higher priority only shown on the console.
+# This creates a logging sink and handler that puts all messages at or above the TRACE level into a logfile for each run.
+logger.add("file_{time}.log", level="TRACE", encoding="utf8") # Unicode instructions needed to avoid file write errors.
+
+@logger.catch # wraps entire function in a trap to display enhanced error tracebaks after an exception occurs.
```
Comment (TheLastGimbus) [inline google_photos_takeout_helper/__main__.py:10]: So we don't need any `try-catch` in `if __name__ == '__main__'` ?

**Comment by TheLastGimbus** [comment]
I tried to add some dummy `raise` in one place, and:
```python
An error has been caught in function '<module>', process 'MainProcess' (10086), thread 'MainThread' (140557777143616):
Traceback (most recent call last):

> File "google_photos_takeout_helper/__main__.py", line 714, in <module>
    main()
    └ <function main at 0x7fd62736ae50>

  File "google_photos_takeout_helper/__main__.py", line 609, in main
    for_all_files_recursive(
    └ <function main.<locals>.for_all_files_recursive at 0x7fd627273160>

  File "google_photos_takeout_helper/__main__.py", line 127, in for_all_files_recursive
    if filter_fun(file):
       │          └ PosixPath('/home/mateusz/Downloads/photos-takeout/mama/Takeout/no_json_found.txt')
       └ <function main.<locals>.<lambda> at 0x7fd627273f70>

  File "google_photos_takeout_helper/__main__.py", line 612, in <lambda>
    filter_fun=lambda f: (is_photo(f) or is_video(f))
                      │   │        │     │        └ PosixPath('/home/mateusz/Downloads/photos-takeout/mama/Takeout/no_json_found.txt')
                      │   │        │     └ <function main.<locals>.is_video at 0x7fd627273280>
                      │   │        └ PosixPath('/home/mateusz/Downloads/photos-takeout/mama/Takeout/no_json_found.txt')
                      │   └ <function main.<locals>.is_photo at 0x7fd6272731f0>
                      └ PosixPath('/home/mateusz/Downloads/photos-takeout/mama/Takeout/no_json_found.txt')

  File "google_photos_takeout_helper/__main__.py", line 155, in is_video
    raise NotADirectoryError

NotADirectoryError
```
Oh wow, this is *really nice* :100:

**Comment by TheLastGimbus** [comment]
This is super pretty, thak you!

**Comment by conradstorz** [comment]
I am very pleased you like it. Loguru has been very helpful in my education with Python.

**Comment by TheLastGimbus** [comment]
Yeah, it helped clean stuff out before progess bar

**Pull request merged**